src/api/java/com/minecolonies/coremod/colony/requestsystem/location/ILocatable.java

```diff
@@ -0,0 +1,16 @@
+package com.minecolonies.coremod.colony.requestsystem.location;
```

Comment:
I think this belong to a different pr, and could be merge well before dman 2.0 is ready